libraries/integration/Microsoft.Bot.Builder.Integration.AspNet.Core/Microsoft.Bot.Builder.Integration.AspNet.Core.csproj

```diff
@@ -46,6 +46,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <ProjectReference Include="..\..\Microsoft.Bot.Builder.Dialogs\Microsoft.Bot.Builder.Dialogs.csproj" />
     <ProjectReference Include="..\..\Microsoft.Bot.Builder\Microsoft.Bot.Builder.csproj" />
```

Comment:
Dialogs bundle Bot.Builder.csproj, we can take out this reference

Reply:
We can move the Dialog dependent code to the Dialogs package and create an extension method there, but if we do that then will need to take a dependency in the Dialog package to Microsoft.AspNetCore.App package, which might seem equally strange as Dialogs may be run outside ASP.NET.